web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.ts

```diff
@@ -102,9 +102,9 @@ export class IngestUiComponent implements OnInit {
 
   onChange() {
     if (this.step.options) {
-      this.step.options.input_file_path = this.folder;
-      this.step.options.transform_param = `entity-name=${this.step.options.targetEntity},flow-name=${this.flow.name}`;
-      this.step.options.output_collections = `${this.step.options.targetEntity}`;
+      this.step.options.inputFilePath = this.folder;
+      // this.step.options.transform_param = `entity-name=${this.step.options.targetEntity},flow-name=${this.flow.name}`;
+      // this.step.options.output_collections = `${this.step.options.targetEntity}`;
       this.saveStep.emit(this.step);
     }
   }
```

Comment:
To avoid conflicts I suggest rollback ingest options. My PR will take care of that.